CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 * [BUG] Ensure node IP is routeable
 * [FEATURE] Reuse generated CA certificate
 * [FEATURE] Ensure xhyve driver uses same IP on restarts
-* [FEATURE] Add defaut insecure registry flag to include minishift service IP range
```

Comment:
Personally I would not change the CHANGELOG in retrospective. This is part of the history.